### PR TITLE
Fix node alias names and deploy services properly

### PR DESCRIPTION
**Pull request by dirkmueller**
The previous scenario file was not deploying services
like requested in the trello card.

### CODE REVIEW

File: scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml [hunk 1]
```diff
@@ -40,8 +40,9 @@ proposals:
     ha:
       storage:
         shared:
-          device: 10.162.66.1:/srv/nfs/cloud4-ha-database
+          device: ##shared_nfs_for_database##
```
Comment (toabctl) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml]: How is this string replaced?

Comment (dirkmueller) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml]: currently by the scenario job, to be cleaned up (see card about unifying jobs with JJB in team green)

**Comment by vmoravec** [comment]
+1

File: scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml (hunk 1 above)
Comment (bmwiedemann) [inline scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml]: actually that was not part of the card when we refined it ;-)

**Pull request merged**